lib/client/services/creativeTemplate/creativeTemplate.type.ts

```diff
@@ -134,7 +151,7 @@ export type CreativeTemplate = {
   /**
    * The description of the creative template. This attribute is optional.
    */
-  description: string;
+  description?: string;
```

Comment:
This has been added for consistency with GAM API [docs](https://developers.google.com/ad-manager/api/reference/v202505/CreativeTemplateService.CreativeTemplate)